client/starwhale/core/model/model.py

```diff
@@ -305,7 +310,30 @@ def eval_user_handler(
             )
 
     def info(self) -> t.Dict[str, t.Any]:
-        return self._get_bundle_info()
+        _manifest = self._get_bundle_info()
+        _manifest["step_spec"] = {}
+        _store = self.store
+        _om = {}
+        if _store.bundle_path.is_dir():
```

Comment:
model bundle_path is file type.

Reply:
remove `_store.bundle_path.is_dir()` check?

Reply:
done